src/js/modules/comments/components/CommentThread.js

```diff
@@ -34,48 +36,74 @@ const styles = {
   },
 };
 
-const CommentThread = ({
-  classes,
-  comments,
-  article,
-  createComment,
-  sessionUser,
-  session,
-}) => {
-  const handleCreateComment = values => {
-    createComment(
+class CommentThread extends React.Component {
+  constructor(props) {
+    super(props);
+  }
+
+  componentDidUpdate(prevProps, prevState) {
+    if (this.props.session) {
```

Comment:
What if the props update but the user is already logged in? You should diff it with the prevProps